primitives/src/proxy_type.rs

```diff
@@ -36,6 +37,11 @@ pub enum ProxyType {
     CancelProxy,
     Governance,
     Staking,
+    CreateMarket,
+    ReportOutcome,
+    ProvideLiquidity,
+    BuySellCompleteSets,
```

Comment:
I think providing liquidity only really makes sense if you can also buy complete sets, so joining these two into one seems fine to me.

Reply:
I think there is a difference. With `ProvideLiquidity`, the account can only manage existing liquidity, i.e. withdraw or deposit in a market. With `BuySellCompleteSets`, the account can actually directly spend tokens and increase liquidity in a market and also start to operate on other markets. It is much more powerful imo.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1038/commits/b80ae75c2da1bc7ce687cb5e2a5aa6d1aef2d37c